src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -317,6 +317,7 @@ void HierRTLMP::hierRTLMacroPlacer()
   //
   block_ = db_->getChip()->getBlock();
   dbu_ = db_->getTech()->getDbUnitsPerMicron();
+  manufacturing_grid_ = db_->getTech()->getManufacturingGrid();
```

Comment:
You should check hasManufacturingGrid() before assigning.

Reply:
fixed